### ISSUE TITLE
How can I convert .pth weight to tflite? 

### ISSUE THREAD

**Issue by HripsimeS** [QuocThangNguyen/deep-metric-learning-tsinghua-dogs#7]
@QuocThangNguyen Hello!

I would like to convert a pytorch (pth) trained model to TensorFlow Lite (tflite) format. Do you have some advice how to do it properly for your Fine-Grained classification models? I saw that in general the converting process has several steps, convert Pytorch to ONNX, ONNX to TensorFlow and then TensorFlow to TensorFlow Lite. But if you know a quick way to do it or maybe there is a special script that can do all these steps without errors, that would be great.

Look forward to hearing from you!

**Comment by QuocThangNguyen** [comment]
Hi @HripsimeS, I mainly work with Pytorch, and have never used Tensorflow or Tensorflow Lite before. But I usually see that the path `pth -> onnx -> tflite` is widely used in the industry. Hope this help.
